Majasdarbs/06-majasdarbs/OlegsKozlovs/app.js

```diff
@@ -0,0 +1,30 @@
+const characters = ["a", "b", "c", "d", "e", "g", "k", "m", "o", "q", "t", "z"];
+
+let arrayId;
+
+let output = document.getElementById("main");
+
+const printValues = () => { output.innerText = characters[arrayId]};
+
+const getValue = () => {
+    arrayId = prompt("Please provide character index:");
+    if (!isNaN(arrayId)) {
```

Comment:
you can combine 3 conditions in a single if.
It is more readable if you have multiple ifs, but in this case you could easily do
if(arrayId >= 0 && arrayId <= characters.length - 1 && !isNaN(arrayId))